packages/api-contract/src/base/Code.ts

```diff
@@ -42,12 +52,12 @@ export class Code<ApiType extends ApiTypes> extends Base<ApiType> {
   constructor (api: ApiBase<ApiType>, abi: string | Record<string, unknown> | Abi, wasm: Uint8Array | string | Buffer | null | undefined, decorateMethod: DecorateMethod<ApiType>) {
     super(api, abi, decorateMethod);
 
-    this.code = isWasm(this.abi.info.source.wasm)
+    this.code = isValidCode(this.abi.info.source.wasm)
```

Comment:
_Slightly_ weird that a different format may come through in `source.wasm`, but either way, all ok.

Reply:
the field in the contract metadata will most likely be renamed once the support for RISC-V is released. not something we should address now imo.